### PR TITLE
Enhance PrestoEngine and ElasticSearchEngine, adatps to Linkis1.0 architecture. 

### PR DESCRIPTION
**Pull request by wForget**
### What is the purpose of the change

Enhance PrestoEngine, adatps to Linkis1.0 architecture. #712 

Enhance ElasticSearchEngine, adatps to Linkis1.0 architecture. #711

### CODE REVIEW

File: linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsConvertResponseException.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.webank.wedatasphere.linkis.engineplugin.elasticsearch.exception
+
+import com.webank.wedatasphere.linkis.common.exception.ErrorException
+
+/**
+ *
+ * @author wang_zh
```
Comment (leeebai) [inline linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsConvertResponseException.scala:22]: The author information needs to be removed.

File: linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsParamsIllegalException.scala [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.webank.wedatasphere.linkis.engineplugin.elasticsearch.exception
+
+import com.webank.wedatasphere.linkis.common.exception.ErrorException
+
+/**
+ *
```
Comment (leeebai) [inline linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsParamsIllegalException.scala:21]: The author information needs to be removed. Same to other place

File: linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/ElasticSearchEngineConnPlugin.scala [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package com.webank.wedatasphere.linkis.engineplugin.elasticsearch
```
Comment (leeebai) [inline linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/ElasticSearchEngineConnPlugin.scala:1]: Each class requires a license file. You do not need to specify webank, but only Apache License
Version 2.0
http://www.apache.org/licenses/LICENSE-2.0

**Review by leeebai — CHANGES_REQUESTED** [review]

File: linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsParamsIllegalException.scala (hunk 2 above)
Comment (wForget) [reply linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsParamsIllegalException.scala:21]: > The author information needs to be removed. Same to other place

Thanks, i will remove it.

File: linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/ElasticSearchEngineConnPlugin.scala (hunk 3 above)
Comment (wForget) [reply linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/ElasticSearchEngineConnPlugin.scala:1]: > Each class requires a license file. You do not need to specify webank, but only Apache License
> Version 2.0
> http://www.apache.org/licenses/LICENSE-2.0

Thanks, I will add the license.

**Pull request closed without merging**